jsapp/js/account/security/mfa/mfaSection.component.tsx

```diff
@@ -163,21 +164,23 @@ export default class SecurityRoute extends React.Component<{}, SecurityState> {
             {t('Two-factor authentication')}
           </bem.SecurityRow__title>
 
-          <bem.SecurityRow__buttons>
-            <ToggleSwitch
-              label={this.state.isMfaActive ? t('Enabled') : t('Disabled')}
-              checked={this.state.isMfaActive}
-              onChange={this.onToggleChange.bind(this)}
-            />
-          </bem.SecurityRow__buttons>
-        </bem.SecurityRow__header>
-
-        <bem.SecurityRow__description>
-          {t(
-            'Two-factor authenication (2FA) verifies your identity using an authenticator application in addition to your usual password. ' +
-              'We recommend enabling two-factor authenication for an additional layer of protection.'
-          )}
-        </bem.SecurityRow__description>
+          <bem.SecurityRow__description>
+            {t(
+              'Two-factor authentication (2FA) verifies your identity using an authenticator application in addition to your usual password. ' +
+                'We recommend enabling two-factor authentication for an additional layer of protection.'
+            )}
```

Comment:
I wonder if this `+` would work with how the strings are extracted for transifex… Maybe it would be better to do this:
```ts
{t(
  'Two-factor authentication (2FA) verifies your identity using an authenticator application in addition to your usual password.'
)}
{t(
  'We recommend enabling two-factor authentication for an additional layer of protection.'
)}            
```